third_party/BUILD

```diff
@@ -237,16 +237,17 @@ extern(
     path = "browser/w3c_selection.js",
 )
 
-# svg.js and webassembly.js are not part of the common externs provided by the
-# js-compiler binary.
+# svg.js is not part of the common externs provided by the js-compiler binary.
 alias(
     name = "svg.js",
     actual = "@com_google_closure_compiler//:contrib/externs/svg.js",
 )
 
+# webassembly.js _is_ in the common externs.zip, but not in the version of
```

Comment:
It says "in the tatest closure-compiler" in commit msg and here you are are saying it doesn't exist in the version we depend on. Did I misunderstand what you mean?

Reply:
Sorry if it was unclear: The `@com_google_closure_compiler` reference pulls straight from master, so webassembly.js is in the `externs/browser/` directory where `extern(...)` expects to find it ... however, `extern(...)` itself pulls instead from `@com_google_javascript_closure_compiler` (note the `_javascript_` in the middle), which is a different dependency. I am not 100% clear on exactly what brings in that dependency (I _think_ it is from jsinterop-generator, but possibly from somewhere further upstream - rules_closure is my best guess), but that is version v20200112, which predates the webassembly.js file moving.

If these aliases read from the same place as extern() it would probably be clearer. Instead though the `@com_google_closure_compiler` http_archive rules specifies that elemental2/jscomp.BUILD will serve as the build file, probably so that these files can be more easily picked out.

It might be "good enough" to synchronize `@com_google_closure_compiler` to somehow read the same version that rules_closure brings in? I'm not sure how to attempt that at this time.

Reply:
Yes, I think we should update the com_google_closure_compiler version instead.